Vue-Basic-Video-Chat/src/Session.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <div id="session" v-on:error="errorHandler">
```

Comment:
Not a huge deal, but `@` is a convenient shorthand for `v-on`, in the same way `:` is a shorthand for `v-bind`